src/wp-admin/css/common.css

```diff
@@ -2061,14 +2061,13 @@ html.wp-toolbar {
 }
 
 /* Replace default marker in details element */
-.metabox-holder details > summary,
```

Comment:
This is having additional unforseen consequences. Mainly, the navigational headings that served as landmarks are now removed from the boxes such as post, categories, and the like. I tried to pinpoint where this was happening but wasn't quite sure where to leave this.

Reply:
@rkingett Is this something that only happened with the change you've linked to, or did it happen before? I'm asking because I'm wondering whether it's actually been caused by my removing the `h2` and `h3` tags in my original commit. (I did that after reading conflicting advice on whether I should or not.)

I just want to be sure where the problem lies so that we know where to fix it.

Reply:
I've just pushed a couple of commits to add back the `h2 and `h3` tags.